frontend/src/components/ReplayButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from "react";
+import styled from "styled-components";
+
+interface ReplayProps {
+  scale?: number,
+}
+
+const ReplayButtonContainer = styled.div`
+  display: flex;
+  padding: 5px;
+  width: max-content;
+`;
+
+const ReplayText = styled.p`
+  color: #000;
+  font-family: "Rubik";
+  font-weight: 700;
+  margin-bottom: 0;
+  margin-right: 5px;
+`;
+
+const ReplayButtonSVG = styled.svg`
+  ${(props) =>
+    `
+    transform: scale(${props.scale});
+  `}
+  path: hover {
+    cursor: pointer;
+  }
+`;
```

Comment:
Do we want this as an svg, or just use an image and scale it?

Reply:
I think this is gonna be an svg but then the svgs exported from figma are kinda big